### PR TITLE
Added recipe for jpype1

### PR DESCRIPTION
**Pull request by pmlandwehr**
Pinging @originell, @marscher, and @cheffe in case they'd like to be added as maintainers to the `jpype1` builder on [conda-forge](http://conda-forge.github.io), a library of community-built [`conda`](http://conda.pydata.org) packages. The maintenance burden here is light: all testing and releases are built and deployed automatically with CI. Changes to how the package is being built can be controlled by changing the recipe which will be located at https://github.com/conda-forge/jpype1-feedstock shortly after this PR is finished and merged. If you aren't interested in helping maintain the build, that's entirely fine too.

### CODE REVIEW

**Comment by originell** [comment]
I'm not interested :) but maybe one of the others is :)

File: recipes/jpype1/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ camelName }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
```
Comment (jakirkham) [inline recipes/jpype1/meta.yaml]: Please add `toolchain`.

**Comment by jakirkham** [comment]
Seem to have a segmentation fault in the builds. Restarted just in case it was due to some transient recipe being converted at the time. Though I'm not too hopeful.

**Comment by marscher** [comment]
#1721

**Comment by jakirkham** [comment]
Thanks @marscher . Closing as a dup. Please add yourself as a maintainer on the feedstock if you haven't already, @pmlandwehr .

**Pull request closed without merging**